pkg/scanner/utils/utils.go

```diff
@@ -64,6 +65,29 @@ func FormatPatchVersion(version string) string {
 		if _, err := strconv.Atoi(part[2]); err == nil {
 			version = strings.Join(part[:3], ".") + "-" + strings.Join(part[3:], ".")
 		}
+	} else {
+		r := regexp.MustCompile(`(?P<Number>^[0-9]+)(?P<PreRelease>[a-z]*.*)`)
+		for i := range part {
+			res := r.FindStringSubmatch(part[i])
+			if res == nil {
+				continue
+			}
+			names := r.SubexpNames()
+			regexGroup := map[string]string{}
+			for j := range names {
+				if names[j] != "" {
+					regexGroup[names[j]] = res[j]
+				}
+			}
```

Comment:
We already handle `res == nil`, so I think we don't need to use `SubexpNames`.

```suggestion
			number := res[1]
			preRelease := res[2]
```

Reply:
@knqyf263  thanks, changes done